packages/amplify-cli/src/__tests__/extensions/amplify-helpers/constants.test.ts

```diff
@@ -0,0 +1,12 @@
+import { amplifyCLIConstants } from '../../../extensions/amplify-helpers/constants';
+
+describe('constants', () => {
+  it('should freeze values', () => {
+    expect(amplifyCLIConstants.AmplifyCLIDirName).toBe('amplify');
+    expect(() => {
+      // @ts-ignore
+      amplifyCLIConstants.AmplifyCLIDirName = 'test';
+    }).toThrow();
```

Comment:
Can you update `toThrow()` to check for a specific error.

Reply:
Sure! I updated to specify the Error object.